### DISCUSSION TITLE
Unable to install on M1 MacBook Air 12.3.1

### DISCUSSION THREAD

**Discussion by SlapsOnly** [Homebrew/discussions#3352] [Getting started]
When running /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

I get the error message
```
Error: Fetching /opt/homebrew/Library/Taps/osx-cross/homebrew-avr failed!
Error: Some taps failed to update!

```
I am advised to run `brew tap --repair `
But brew isn't installed. 

Plz help.
Entire terminal text is below. 

user@USER-MacBook-Air ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R user:admin /opt/homebrew
==> Downloading and installing Homebrew...
HEAD is now at ad13524bc Merge pull request #13402 from samford/livecheck/update-default-match_data
fatal: couldn't find remote ref refs/heads/master
Error: Fetching /opt/homebrew/Library/Taps/osx-cross/homebrew-avr failed!
Error: Some taps failed to update!
The following taps can not read their remote branches:
  osx-cross/avr
This is happening because the remote branch was renamed or deleted.
Reset taps to point to the correct remote branches by running `brew tap --repair`
Failed during: /opt/homebrew/bin/brew update --force --quiet
user@USER-MacBook-Air ~ % brew tap --repair
zsh: command not found: brew
user@USER-MacBook-Air ~ %

**Comment by SMillerDev** [comment]
Try removing that folder

**Reply by SlapsOnly** [reply]
Remove what folder?

**Reply by SMillerDev** [reply]
The `/opt/homebrew/Library/Taps/osx-cross/homebrew-avr` in the error.
